apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,17 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// +optional
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
+	// +optional
```

Comment:
I find this style in the configuration API a bit confusing.

@mimowo @tenzen-y 

Why can't we assume that if someone specifies a LocalQueueSelector than they really want this to be enabled?

Otherwise we should validate that both enable is true and LocalQueueSelector. But god forbid what happens if someone has Enable = false but sets a LocalQueueSelector?

Reply:
I think this style is helpful to temporaritly disable the feature without changing the configuration. Otherwise the admin needs to backup the configuration and restore when re-enabling.